frontend/src/components/CaseSVGInfo/index.tsx

```diff
@@ -0,0 +1,29 @@
+import client from '@/src/middleware/apollo-client';
+import { useQuery } from '@apollo/client'
+import GetCaseById from '@/queries/getCaseById';
+import SVGTitle from '../SVGTitle';
+import SVGText from '../SVGText';
+import styles from './casesvginfo.module.css';
+
+interface CaseSVGInfoProps {
+    selectedCaseId: string;
+}
+
+export default function CaseSVGInfo({ selectedCaseId }: CaseSVGInfoProps) {
+    const { data } = useQuery(GetCaseById, { client, variables: { id: selectedCaseId }});
+    if (!data) {
+        return null;
+    }
```

Comment:
We are doing it in many places, can we made some checking function for it, so it would look better?